### PR TITLE
clumsy: Add version 0.2

### PR DESCRIPTION
**Pull request by sebmaynard**
clumsy makes your network condition on Windows significantly worse, but in a managed and interactive manner. 

Fantastic for network testing games or other software

### CODE REVIEW

File: bucket/clumsy.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/jagt/clumsy/releases/download/$versionclumsy-$version-win32.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/jagt/clumsy/releases/download/$version/clumsy-$version-win32.zip"
+            }
+        }
+    }
+}
```
Comment (Ash258) [inline bucket/clumsy.json:40]:
```suggestion
{
    "version": "0.2",
    "description": "Broken network simulator",
    "homepage": "https://jagt.github.io/clumsy/",
    "license": "MIT",
    "architecture": {
        "64bit": {
            "url": "https://github.com/jagt/clumsy/releases/download/0.2/clumsy-0.2-win64.zip",
            "hash": "19042ae5e28412a8eb2dc67f4bc3b606ef04cb6f46ef72563f25e41b2bc67609"
        },
        "32bit": {
            "url": "https://github.com/jagt/clumsy/releases/download/0.2/clumsy-0.2-win32.zip",
            "hash": "ad79829832996024981d0161d4862cf28f0d7839a6c6b1b81b55d6335a1b4df9"
        }
    },
    "bin": "clumsy.exe",
    "shortcuts": [
        [
            "clumsy.exe",
            "clumsy"
        ]
    ],
    "persist": "config.txt",
    "checkver": {
        "github": "https://github.com/jagt/clumsy"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/jagt/clumsy/releases/download/$version/clumsy-$version-win64.zip"
            },
            "32bit": {
                "url": "https://github.com/jagt/clumsy/releases/download/$version/clumsy-$version-win32.zip"
            }
        }
    }
}

```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by sebmaynard** [comment]
Thank you for the suggestions, still learning how these things should work!

File: bucket/clumsy.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/jagt/clumsy/releases/download/$version/clumsy-$version-win64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/jagt/clumsy/releases/download/$version/clumsy-$version-win32.zip"
+            }
+        }
```
Comment (Ash258) [inline bucket/clumsy.json:35]:
```suggestion
        },
        "hash": {
            "url": "https://jagt.github.io/clumsy/download.html",
            "regex": "$basename</a>\\s*\\(MD5:$md5\\)"
        }
```

File: bucket/clumsy.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "homepage": "https://jagt.github.io/clumsy/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/jagt/clumsy/releases/download/0.2/clumsy-0.2-win64.zip",
+            "hash": "19042ae5e28412a8eb2dc67f4bc3b606ef04cb6f46ef72563f25e41b2bc67609"
+        },
+        "32bit": {
+            "url": "https://github.com/jagt/clumsy/releases/download/0.2/clumsy-0.2-win32.zip",
+            "hash": "ad79829832996024981d0161d4862cf28f0d7839a6c6b1b81b55d6335a1b4df9"
```
Comment (Ash258) [inline bucket/clumsy.json:13]:
```suggestion
            "hash": "md5:9aab0d257661a4f75831a7186254725b"
```

File: bucket/clumsy.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "0.2",
+    "description": "Broken network simulator",
+    "homepage": "https://jagt.github.io/clumsy/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/jagt/clumsy/releases/download/0.2/clumsy-0.2-win64.zip",
+            "hash": "19042ae5e28412a8eb2dc67f4bc3b606ef04cb6f46ef72563f25e41b2bc67609"
```
Comment (Ash258) [inline bucket/clumsy.json:9]:
```suggestion
            "hash": "md5:c5117edad320930d14d18c1cac2a4ccd"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Pull request merged**